main.tf

```diff
@@ -3,6 +3,23 @@ locals {
     managed_by = "terraform"
     module     = "materialize"
   })
+
+  # Disk support configuration
+  disk_config = {
+    install_openebs           = var.enable_disk_support ? lookup(var.disk_support_config, "install_openebs", true) : false
+    run_disk_setup_script     = var.enable_disk_support ? lookup(var.disk_support_config, "run_disk_setup_script", true) : false
+    local_ssd_count           = lookup(var.disk_support_config, "local_ssd_count", 1)
+    create_storage_class      = var.enable_disk_support ? lookup(var.disk_support_config, "create_storage_class", true) : false
```

Comment:
Is there a time when we would not create the storage_class and still enable disk support? It seems like we'd need both of these right?